src/components/Search/Filters/FilterButton.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactElement, useEffect, useState } from 'react'
+import Box from '../../atoms/Box'
+import FilterOptions, { sortDirectionOptions, TypeKeys } from './FilterOptions'
+import styles from './FilterButton.module.css'
+import { useRouter } from 'next/router'
+import { SortDirectionOptions } from '../../../models/SortAndFilters'
+
+export interface Option {
+  display: string
```

Comment:
suggestion(blocking): I think we can make `display` an optional prop. If nothing is provided use the `value` string per default. I like the idea of introducing different props for display and value, but in this case it seems unecessary to have it be mandatory.